data/menus.yaml

```
@@ -30,6 +30,7 @@
     - Well-founded recursion: extras/well_founded_recursion.html
     - Speeding up Lean files: extras/speedup.html
     - Pitfalls and common mistakes: extras/pitfalls.html
+    - Did I really prove it?: data/did_you_prove_it.md
```

```suggestion
    - Did you really prove it?: data/did_you_prove_it.md
```
only to match the name of the file.